src/js/pages/Arbitration/index.jsx

```diff
@@ -124,66 +146,89 @@ class Arbitration extends Component {
           </CheckButton>
         </ButtonGroup>
         <p className="text-center">
-          <Button color="primary" onClick={this.displayDialog(true)} disabled={selectedUser === null}>Resolve
-            dispute</Button>
+          <Button color="primary" onClick={this.displayDialog(true)} disabled={selectedUser === null}>
+            {t('arbitration.resolve')}
+          </Button>
         </p>
       </ModalBody>
       <ModalFooter>
-        <Button onClick={this.handleClose}>Cancel</Button>
+        <Button onClick={this.handleClose}>{t('general.cancel')}</Button>
       </ModalFooter>
     </Modal>);
   }
 
   render() {
-    const {escrow, address, loading, buyerInfo, sellerInfo} = this.props;
+    const {t, escrow, address, loading, buyerInfo, sellerInfo} = this.props;
 
-    if(!escrow || !buyerInfo || !sellerInfo){
+    if (!escrow || !buyerInfo || !sellerInfo) {
       return <Loading/>;
     }
 
-    if(addressCompare(escrow.buyer, address) || addressCompare(escrow.seller, address)) return <ErrorInformation message="You cannot arbitrate your own disputes"/>;
-    if(!addressCompare(escrow.arbitrator, address)) return <ErrorInformation message="You are not the arbitrator of this dispute"/>;
+    if (addressCompare(escrow.buyer, address) || addressCompare(escrow.seller, address)) {
+      return <ErrorInformation message={t('arbitration.ownDispute')}/>;
+    }
+    if (!addressCompare(escrow.arbitrator, address)) {
+      return <ErrorInformation message={t('arbitration.notYours')}/>;
+    }
 
-    if(loading) return <Loading mining={true} />;
+    if (loading) {
+      return <Loading mining={true}/>;
+    }
 
-    const status = getArbitrationStatus(escrow.arbitration.result);
+    const status = this.getArbitrationStatus(escrow.arbitration.result);
     const openedByBuyer = addressCompare(escrow.arbitration.openBy, escrow.buyer);
     return (
       <div className="escrow">
-        <h2>Dispute Details <span className={"arbitrationStatus " + status}>{status}</span></h2>
+        <h2>
+          <Trans i18nKey="arbitration.disputeDetails" values={{status}}>
+            Dispute Details <span className={"arbitrationStatus " + status}>{{status}}</span>
```

Comment:
:heavy_multiplication_x:

Reply:
This is a special case where the `<Trans>` component does the magic. This let's us use tags inside the translations.